cmd/earth/main.go

```diff
@@ -387,6 +395,72 @@ func newEarthApp(ctx context.Context, console conslogging.ConsoleLogger) *earthA
 				},
 			},
 		},
+		{
+			Name:   "org",
+			Usage:  "Earthly organization administration",
+			Hidden: true,
+			Subcommands: []*cli.Command{
+				{
+					Name:   "create",
+					Action: app.actionOrgCreate,
+				},
+			},
+		},
+		{
+			Name:        "secrets",
+			Usage:       "Earthly secrets",
+			Description: "Access and modify secrets",
+			Hidden:      true,
+			Subcommands: []*cli.Command{
+				{
+					Name:   "get",
+					Action: app.actionSecretsGet,
+					Flags: []cli.Flag{
+						&cli.BoolFlag{
+							Aliases:     []string{"n"},
+							Usage:       "disable newline at the end of the secret",
+							Destination: &app.disableNewLine,
+						},
+					},
+				},
+				{
+					Name:   "set",
+					Action: app.actionSecretsSet,
+					// TODO expand flags to allow reading secret from stdin or a file
+				},
+			},
+		},
+		{
+			Name:        "register",
+			Usage:       "Register for an earthly account",
+			Description: "Register for an earthly account",
+			Hidden:      true,
+			Action:      app.actionRegister,
+			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:        "email",
+					Usage:       "email address",
+					Destination: &app.email,
+				},
+				&cli.StringFlag{
+					Name:        "token",
+					Usage:       "email verification token",
```

Comment:
Forgot about these ones

Reply:
whoops, thanks for pointing that out. fixed.